lib/services/storageManagement2/package.json

```diff
@@ -11,7 +11,7 @@
     "Wang, Yugang <yugangw@microsoft.com>",
     "Zavery, Amar <amzavery@microsoft.com>"
   ],
-  "version": "0.12.0",
+  "version": "0.12.0-alpha",
```

Comment:
I think it should be "0.12.0-preview", as it is a common pattern in the node sdk. I have submitted another PR https://github.com/Azure/azure-sdk-for-node/pull/1595 as I had published new version of the client runtime and wanted to regenerate using a new version of the code generator.